docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` value after submiting the same name. 
```

Comment:
Would "The Service account name on the Workload Details Screen is no longer reset to `default` when re-submitting the currently defined name." make sense?